src/test/ui/conditional-compilation/cfg_accessible-not_sure.rs

```diff
@@ -0,0 +1,85 @@
+// revisions: edition2015 edition2021
+// [edition2015]compile-flags: --edition=2015
+// [edition2021]compile-flags: --edition=2021
+
+#![feature(extern_types)]
+#![feature(cfg_accessible)]
+
+// Struct::unresolved - error
+
+struct Struct {
+    a: u8,
+    pub b: u8
```

Comment:
Not much difference between pub and non-pub here, since the check in the same module.
It can either be removed as redundant or moved to a separate module to make difference.